policyengine_us/variables/gov/states/mt/tax/income/exemptions/mt_interest.py

```diff
@@ -0,0 +1,26 @@
+from policyengine_us.model_api import *
+
+
+class mt_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montana exemptions amount"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2022/12/Form-2-2022-Instructions.pdf#page=25"
+    defined_for = StateCode.MT
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.mt.tax.income.exemptions
+        filing_status = tax_unit("filing_status", period)
+        cap_amount = p.amount[filing_status]
+        dividend_income = tax_unit(
+            "taxsim_dividends", period
```

Comment:
```suggestion
            "taxsim_intrec", period
```

Reply:
taxsim variables are for validation only; do not use them for our computations. we have other interest variables